.codeclimate.yml

```diff
@@ -64,14 +64,6 @@ plugins:
     exclude_paths:
       - "decidim_app-design/**/*"
 
-  markdownlint:
-    enabled: true
-
-    exclude_paths:
-      - docs/*
-      - .github/*
-      - CHANGELOG.md
```

Comment:
I see now how all those offenses crept in. I didn't enable `mdl` on some paths because I was lazy and I thought we could enable it bit by bit :)

Reply:
LOL so @josepjaume what about removing these exclusions and keep with the current `mdl` linter? 😄

Reply:
But then it won't seamlessly integrate with Visual Code! Hahaha. I think I'll just close this PR, your points are valid, and this was actually my goal 🤣